metadrive/obs/top_down_obs_multi_channel.py

```diff
@@ -142,7 +165,13 @@ def draw_scene(self):
         ego_heading = vehicle.heading_theta
         ego_heading = ego_heading if abs(ego_heading) > 2 * np.pi / 180 else 0
 
-        for v in self.engine.traffic_manager.vehicles:
+        vehicles = []
+        if hasattr(self.engine, "traffic_manager"):
+            vehicles = self.engine.traffic_manager.vehicles
+        elif hasattr(self.engine, "scenario_traffic_manager"):
+            vehicles = self.engine.scenario_traffic_manager.vehicles
```

Comment:
As I said before, let's draw all other things besides vehicles. A suggestion here is that we can actually cancel this `if` condition. For both PGEnvironment and ScenarioEnvironment, all objects are actually in the `engine._spawned_objects`. What a manager does is a set of rules to add, delete, or actuate objects. Thus we can access all objects through env.engine directly, regardless of what the traffic manager is.

Reply:
Done!